pageserver/src/tenant.rs

```diff
@@ -3082,6 +2938,21 @@ impl Tenant {
         Ok(uninit_mark)
     }
 
+    fn create_timeline_delete_mark(&self, timeline_id: TimelineId) -> anyhow::Result<()> {
+        let marker_path = self
+            .conf
+            .timeline_delete_mark_file_path(self.tenant_id, timeline_id);
+
+        // Note: we're ok to replace existing file.
+        let _ = std::fs::OpenOptions::new()
+            .write(true)
+            .create(true)
+            .open(&marker_path)
```

Comment:
```suggestion
        std::fs::OpenOptions::new()
            .write(true)
            .create(true)
            .open(&marker_path)
            .map(|_| ())
            .or_else(|e| if e.kind() == std::io::ErrorKind::AlreadyExists { Ok(()) } else { Err(e) })
```

Or does the AlreadyExists have some portability issues or unreliable or something?

Reply:
Oops, I guess I again misread `create(true)` to mean `create_new(true)` so what exactly is the error we want to ignore here...? None, yeah, originally I misread this as ignoring an error but it's just about the drop.

Nevermind, this was just noise, resolving.